scripts/run_travis.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -eu
+
+# rebase on current upstream master, so that we run current tests
+git remote get-url upstream >/dev/null 2>&1 || git remote add upstream https://github.com/rhinstaller/kickstart-tests
+git fetch upstream
+git rebase upstream/master
+
+# list of tests that are changed by the current PR
+TESTS=$(git diff --name-only upstream/master..HEAD -- *.ks.in *.sh | sed 's/\.ks\.in$//; s/\.sh$//' | sort -u)
+
+# if the PR changes anything in the test runner, or does not touch any tests, pick a few representative tests
+if [ -z "$TESTS" ] || [ -n "$(git diff --name-only upstream/master..HEAD -- containers scripts)" ]; then
+    TESTS="$TESTS
+bindtomac-network-device-default-httpks
+container
+lvm-1
+network-device-mac-httpks
+network-static
```

Comment:
We have a test group called `coverage`, but there are about thirty tests at this moment. Maybe we could create a new group called `minimal` that would provide a reasonable subset for Travis.

Reply:
I added a FIXME comment as a reminder.